### ISSUE TITLE
It might be more clear to use 'satoshis' instead of 'Bitcoin' for value units

### ISSUE THREAD

**Issue by billygarrison** [renepickhardt/The-Lightning-Network-Book#17]
I've noticed a few different uses of Bitcoin throughout the book so far as unit of value (e.g. "how many Bitcoins" and "how much Bitcoin"). At one point I changed it to BTC to clear it up, but that introduces an inconsistency as well.

I propose that you use 'satoshis' as the unit of value instead of 'Bitcoin' because a) that is the real unit that the protocol uses, and b) it is clear with no ambiguity.

**Comment by billygarrison** [comment]
On second thought, I don't think this necessarily works for all occurrences of 'Bitcoins' (e.g. it would be kind of weird to see "The coinbase transaction generates new satoshis"). But might still be useful when describing funding transactions, channel balances, commitment transactions, etc.

**Comment by renepickhardt** [comment]
I kind of like your proposal but I am also critical about it. The main reason against this is the fact that Bitcoin is the term most people will have heard and we are here to scale bitcoin and make a payment system on top of bitcoin. Also we would have to consider using millisatoshi as this is actually the unit that is being used within the protocol which makes it almost absurd.

**Comment by billygarrison** [comment]
I agree with you. I guess the main take-away for me is that we should be careful with saying things along the lines of "how many Bitcoins are in the channel?" since it kind of implies whole Bitcoins are in the channel. Perhaps in that case "how much Bitcoin is in the channel?" is better.
